server/routes/users.js

```diff
@@ -5,10 +5,46 @@ const validateID = require("../middleware/validateID");
 const { authenticateAccessToken, authenticateRefreshToken } = require("../middleware/authenticateToken");
 const User = require("../models/user");
 const router = express.Router();
+const { ObjectId } = require('mongoose').Types;
 
 router.get("/", async (req, res) => {
-    const users = await User.find().sort("userName");
-    res.send(users);
+    const { userName, email, role, _id } = req.query;
```

Comment:
this isnt the right way to do it. you shouldnt be adding any query into the `/` route . its also extremely unsafe to do this. we already have a get `/:id` so if you need to get the user details you should be using that

Reply:
you should never be passing a query to / route. thats literally just for getting users

Reply:
Undid this change and created a skills get route using the userId the same way as done in Resume.jsx (which I know we want to change at some point)